packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -37,7 +37,7 @@ import {
  */
 export class ZipkinExporter implements SpanExporter {
   static readonly DEFAULT_URL = 'http://localhost:9411/api/v2/spans';
-
+  static readonly OT_HEADER_NAME = 'x-opentelemetry-outgoing-request';
```

Comment:
I'd suggest to do `export const OT_REQUEST_HEADER = 'x-opentelemetry-outgoing-request';` outside the `ZipkinExporter ` class. It will be available for `utils.ts` (no need to declare it again) and in tests.